realtime-telemetry-plugin.js

```diff
@@ -20,22 +18,11 @@ function RealtimeTelemetryPlugin() {
                 return domainObject.type === 'example.telemetry';
             },
             subscribe: function (domainObject, callback, options) {
```

Comment:
@larkin The options argument is no longer needed since the recent changes to the telemetry API doesn't require it.